annotations/annotations_rw.go

```diff
@@ -72,7 +79,7 @@ func (rw *annotationsRW) Read(ctx context.Context, contentUUID string) (*Annotat
 	case http.StatusNotFound:
 		return nil, "", false, nil
 	default:
-		return nil, "", false, fmt.Errorf("annotations RW returned an unexpected HTTP status code in read operation: %v", resp.StatusCode)
+		return nil, "", false, fmt.Errorf("status %v: %w", resp.StatusCode, ErrUnexpectedStatusRead)
```

Comment:
Let use `%d` for the status code.

Reply:
In the original code, it was %v, that's why I tried to preserve it.